installer/installation-notes.md

```diff
@@ -0,0 +1,135 @@
+# Installing the RSP
+
+## Preparation
+
+  * Create a working dir.
+  * git clone https://github.com/lsst-sqre/phalanx into it (using https
+    rather than git+ssh is important).
+  * cd phalanx/installer
+
+## Create installation environment
+
+  * If you don't have a virtualenv for the installer already,
+    `mkvirtualenv -p $(which python3) -r requirements.txt installer`
+    (adjust if you use pyenv or raw virtualenv or whatever; the
+    important thing is to pick up the requirements.txt from the
+    installer directory)
+  * Get the correct write token for this instance's Vault enclave out of
+    1Password
+
+## Prepare for the ingress
+
+There are three cases (thus far) for what the top-level ingress looks
+like.
+
+  * It is externally managed, as are any required TLS certificates.
+    The installer doesn't cope with this case very well yet, and you
+    will need to manually inject the TLS certificate into the squareone
+    configuration post-installation.
+
+  * For each of the second and third, which are the cases the installer
+    script is designed to support, the RSP manages the top-level
+    ingress, and Let's Encrypt is used to manage TLS certificates.  The
+    difference between these cases is that one has a pre-defined fixed
+    IP address (e.g. the T&S sites), and one uses whatever the
+    Kubernetes provider gives up (e.g. GKE).
```

Comment:
The IDF installations use fixed IP addresses, so this isn't any GKE, just the test/dev environments like minikube and red-five. Er, but I see that theIP addresses aren't pinned properly in the ingress-nginx configuration for anything other than dev. We should fix that.